packages/data-models/src/annotations/assertion.js

```diff
@@ -0,0 +1,51 @@
+import User from '@dmodels/user-identity/user.js'
+import IRIProvider from '@dmodels/iri/iri-provider.js'
+
+export default class Assertion {
+  /**
+   * @param {User} author - A user who made an assertion.
+   * @param {object} options - Optional parameters.
+   * @param {number} [options.confidence] - A level of confidence, a whole number.
+   * @param {Date} [options.dateTime] - When an assertion was made.
+   */
+  constructor (author, { confidence = 1, dateTime = new Date() } = {}) {
+    if (!confidence) { throw new Error(Assertion.errMsgs.CONFIDENCE_IS_MISSING) }
```

Comment:
(I suggested a change to make confidence optional in the schema, and it seems like we should be able to automatically validate based upon the schema to avoid needing to keep the code in sync with schema changes)

Reply:
That's a good idea, I'll check which one will work best for us.